packages/core/src/util/sqs.ts

```diff
@@ -1,3 +1,3 @@
-export const MAX_SQS_MESSAGE_SIZE = 256000;
-export const SQS_MESSAGE_BATCH_SIZE = 100;
+export const MAX_SQS_MESSAGE_SIZE = 246000;
+export const SQS_MESSAGE_BATCH_SIZE = 250;
```

Comment:
This seem dangerous, no? We're using this constant to determine how many requests in parallel we can send to SQS, so I'm concerned it could lead to rate limits.

Why do we want to update this?

Reply:
"this" being the update on SQS_MESSAGE_BATCH_SIZE